### PR TITLE
Add action to run amp validation

### PR DESCRIPTION
**Pull request by chris-hinds**
Resolves #NUMBER

**Overall change:**
- Add new GH Action to run our AMP validator script
- Run the Simorgh local server in the background in order to run the amp validator script
- At the moment if AMP validation fails it does not fail the build (this is intended)

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by andrewscfc — APPROVED** [review]
👍 What actually stops it failing the build?

**Comment by chris-hinds** [comment]
> 👍 What actually stops it failing the build?

Nothing right now, as it doesn't currently fail the build in Jenkins (because our AMP validation fails). We will want to add. anon 0 exit code to the amp validator script when we do want it to fail the build

File: .github/workflows/simorgh-amp-validator.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: Simorgh CI - AMP Validation
```
Comment (karinathomasbbc) [inline .github/workflows/simorgh-amp-validator.yml]: Could we add bbc-a11y to this too?

```suggestion
name: Simorgh CI - AMP Validation & bbc-a11y
```

File: .github/workflows/simorgh-amp-validator.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install & Build Simorgh
+        run: |
+          npm ci
+          npm run build:local
+      - name: Start Simorgh Server
+        run: nohup node build/server.js > /dev/null 2>&1 &
```
Comment (karinathomasbbc) [inline .github/workflows/simorgh-amp-validator.yml]: Is this an alternative to `npm run dev`?

File: .github/workflows/simorgh-amp-validator.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      - name: Run AMP Validator
+        run: npm run amp:validate
```
Comment (karinathomasbbc) [inline .github/workflows/simorgh-amp-validator.yml]: We could include bbc-ally here
```suggestion
      - name: Run AMP Validator
        run: npm run amp:validate
      - name: Run bbc-a11y
        run: npm run bbc-a11y
```

**Review by karinathomasbbc — APPROVED** [review]

**Comment by karinathomasbbc** [comment]
> > 👍 What actually stops it failing the build?
> 
> Nothing right now, as it doesn't currently fail the build in Jenkins (because our AMP validation fails). We will want to add. anon 0 exit code to the amp validator script when we do want it to fail the build

Let's wait until after the Emotion migration (which may resolve https://github.com/bbc/simorgh/issues/8104 anyway)

File: .github/workflows/simorgh-amp-validator.yml (hunk 2 above)
Comment (chris-hinds) [reply .github/workflows/simorgh-amp-validator.yml]: nope, this starts. the node server as a background process and runs a local production build not the dev server

**Pull request merged**